app/controllers/api/class_members_controller.rb

```diff
@@ -20,9 +20,16 @@ def index
     end
 
     def create
-      student_ids = [class_member_params[:student_id]]
-      students = SchoolStudent::List.call(school: @school, token: current_user.token, student_ids:)
-      result = ClassMember::Create.call(school_class: @school_class, students: students[:school_students])
+      user_ids = [class_member_params[:user_id]]
+      user_type = class_member_params[:type]
+      if %w[teacher owner].include?(user_type)
```

Comment:
just wondering...if we eventually have owners that aren't teachers, would we want them as class members I wonder? the permissions currently don't lend themselves to that iirc (happy to be corrected though)

Reply:
Hmmm yes, that's a good point. I think as long as we're definitely sending `teacher` rather than `owner` from the frontend in the case where the user is both an owner and teacher then I think we can safely get rid of the owner option here